onnx/onnx.proto

```diff
@@ -195,14 +243,23 @@ message TensorProto {
   // raw bytes field. The only exception is the string case, where one is
   // required to store the content in the repeated bytes string_data field.
   //
-  // When this raw_data field is used to store tensor value, it is expected
-  // to use the little-endian order. For floating data types, they should
-  // be of the IEEE 754 format. Boolean type is stored as bytes.
+  // When this raw_data field is used to store tensor value, elements MUST 
+  // be stored in as fixed-width, little-endian order. 
+  // Floating-point data types MUST be stored in IEEE 754 format. 
+  // Complex64 elements must be written as two consecutive FLOAT values, real component first.
+  // Complex128 elements must be written as two consecutive DOUBLE values, real component first.
+  // Boolean type MUST be written one byte per tensor element (00000001 for true, 00000000 for false).
```

Comment:
BOOL should be stored in int32_data above (see comment there)

Reply:
That is consistent with the spec and is explicitly called out as a supported DataType on int32_data. . 

The use of raw_data is optional but supported for all data types except for string. The comment text simply makes it unambiguous how things get encoded when the raw_data field is used.